### ISSUE TITLE
TypeError: Cannot read properties of undefined

### ISSUE THREAD

**Issue by jayvdb** [jslicense/licensee.js#93]
I have `"licensee": "jslicense/licensee.js",` in my devDependencies to use `main` of this repo

I have a .licensee.json which does not have a blueOak defined, as I dont want to use that.

When I run with arg `--osi` I get:

```
pnpm licensee --osi
/home/jayvdb/work/ros-frontend/node_modules/.pnpm/licensee@https+++codeload.github.com+jslicense+licensee.js+tar.gz+cdf5ad12d722b351b41018836029012c2e9b3460/node_modules/licensee/index.js:62
          name.toLowerCase() === configuration.licenses.blueOak.toLowerCase()
                                                                ^

TypeError: Cannot read properties of undefined (reading 'toLowerCase')
    at /home/jayvdb/work/ros-frontend/node_modules/.pnpm/licensee@https+++codeload.github.com+jslicense+licensee.js+tar.gz+cdf5ad12d722b351b41018836029012c2e9b3460/node_modules/licensee/index.js:62:65
    at Array.some (<anonymous>)
    at validConfiguration (/home/jayvdb/work/ros-frontend/node_modules/.pnpm/licensee@https+++codeload.github.com+jslicense+licensee.js+tar.gz+cdf5ad12d722b351b41018836029012c2e9b3460/node_modules/licensee/index.js:61:21)
    at licensee (/home/jayvdb/work/ros-frontend/node_modules/.pnpm/licensee@https+++codeload.github.com+jslicense+licensee.js+tar.gz+cdf5ad12d722b351b41018836029012c2e9b3460/node_modules/licensee/index.js:14:8)
    at checkDependencies (/home/jayvdb/work/ros-frontend/node_modules/.pnpm/licensee@https+++codeload.github.com+jslicense+licensee.js+tar.gz+cdf5ad12d722b351b41018836029012c2e9b3460/node_modules/licensee/licensee:119:16)
    at Object.<anonymous> (/home/jayvdb/work/ros-frontend/node_modules/.pnpm/licensee@https+++codeload.github.com+jslicense+licensee.js+tar.gz+cdf5ad12d722b351b41018836029012c2e9b3460/node_modules/licensee/licensee:87:3)
    at Module._compile (node:internal/modules/cjs/loader:1364:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
    at Module.load (node:internal/modules/cjs/loader:1203:32)
    at Module._load (node:internal/modules/cjs/loader:1019:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)
    at node:internal/main/run_main_module:28:49

Node.js v18.20.4
```

Even if I add a licenses.blueOak to my config, I still see the error

**Comment by kemitchell** [comment]
Do you have `blueOak` literally set to `undefined` in your JSON configuration file? Delete it.

**Comment by kemitchell** [comment]
Hold up. I've been able to replicate.

**Comment by kemitchell** [comment]
Fix released in 10.0.1.
